CMakeLists.txt

```diff
@@ -35,10 +35,8 @@ option(REMOVE_UNUSED_EXTERNALS "enable this to remove those projects from disk"
 option(WITH_EIGEN		"required c++ matrix library.  only disable if you have it already."    ON)
 option(WITH_SWIGMAKE    "helper tools to build python & MATLAB wrappers for C++ libraries with Eigen" ON)
 if (WIN32)
-   if (CMAKE_SIZEOF_VOID_P EQUAL 8)  # gtk only compiles on win64 right now
 	option(WITH_LCM 		"interprocess communications protocol for visualizers, etc" ON)
```

Comment:
can you please move this line (and the matching line in the else) out of the conditional?  much better to have it only in one place.